app/assets/javascripts/darkswarm/directives/country_selector.js.coffee

```diff
@@ -0,0 +1,11 @@
+Darkswarm.directive "countrySelector", () ->
```

Comment:
I think I couldn't get it to work that way. If I recall, the angular script couldn't access the Spree::Config and the indexing of the array was strange.